bin/setup.ps1

```diff
@@ -1,11 +1,14 @@
+# Fail fastp
```

Comment:
Is the `p` supposed to be there? If so, I don't know what fastp is, but maybe it's a Windows thing.

Reply:
Done